Error running application

After downloading the last release (0.5.0-darwin), when I try to run the helper, I get this error:
```
zsh 🌀 ./lcm-0.5.0-darwin --help
panic: html/template: pattern matches no files: `templates/*`

goroutine 1 [running]:
html/template.Must(0x0, 0x278a000, 0xc00044a0c0, 0x278a000)
	/home/travis/.gimme/versions/go1.13.5.linux.amd64/src/html/template/template.go:372 +0x54
github.com/arminc/k8s-platform-lcm/internal.init()
	/home/travis/gopath/src/github.com/arminc/k8s-platform-lcm/internal/server.go:27 +0x58
```
OS: macOS 10.14.6 Mojave
GO: go version go1.13.8 darwin/amd64

--- a/.gitignore
+++ b/.gitignore
@@ -2,3 +2,4 @@ config.yaml
 cmd/fake
 todo.md
 .vscode/**
+dist

--- a/build.sh
+++ b/build.sh
@@ -2,8 +2,8 @@
 
 mkdir dist
 export GOARCH=amd64
-export GOOS=darwin && go build -ldflags="-s -w -X main.Version=$TRAVIS_TAG" -o dist/lcm-$TRAVIS_TAG-darwin cmd/lcm/main.go
-export GOOS=linux && go build -ldflags="-s -w -X cmd.lcm.main.Version=$TRAVIS_TAG" -o dist/lcm-$TRAVIS_TAG-linux cmd/lcm/main.go
-export GOOS=windows && go build -ldflags="-s -w -X cmd.lcm.main.Version=$TRAVIS_TAG" -o dist/lcm-$TRAVIS_TAG-windows cmd/lcm/main.go
+export GOOS=darwin && go build -ldflags="-s -w -X main.Version=${TRAVIS_TAG}" -o dist/lcm-"${TRAVIS_TAG}"-darwin cmd/lcm/main.go
+export GOOS=linux && go build -ldflags="-s -w -X cmd.lcm.main.Version=${TRAVIS_TAG}" -o dist/lcm-"${TRAVIS_TAG}"-linux cmd/lcm/main.go
+export GOOS=windows && go build -ldflags="-s -w -X cmd.lcm.main.Version=${TRAVIS_TAG}" -o dist/lcm-"${TRAVIS_TAG}"-windows cmd/lcm/main.go
 
 ls -la dist

--- a/internal/server.go
+++ b/internal/server.go
@@ -24,7 +24,6 @@ type WebData struct {
 
 var (
 	WebDataVar = WebData{}
-	templates  = template.Must(template.ParseGlob("templates/*"))
 )
 
 func StartServer() {
@@ -75,6 +74,7 @@ func StartServer() {
 }
 
 func index(w http.ResponseWriter, req *http.Request) {
+	templates := template.Must(template.ParseGlob("templates/*"))
 	err := templates.ExecuteTemplate(w, "index.gohtml", WebDataVar)
 	if err != nil {
 		log.WithError(err).Error("Could not server index template")

